### PR TITLE
DOMA-2886 B2B apps front-end

### PR DESCRIPTION
**Pull request by SavelevMatthew**

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]

**Review by gegius — APPROVED** [review]

File: apps/condo/domains/acquiring/components/Alerts/NoConnectedBillings.tsx [hunk 1]
```diff
@@ -9,8 +9,8 @@ const ALERT_BOX_STYLE: CSSProperties = {
 
 const AlertBody: React.FC = () => {
     const intl = useIntl()
-    const AlertMessage = intl.formatMessage({ id: 'services.acquiring.NoBillings.message' })
-    const AlertLinkMessage = intl.formatMessage({ id: 'services.acquiring.NoBillings.link' })
+    const AlertMessage = intl.formatMessage({ id: 'miniapps.acquiring.NoBillings.message' })
```
[the remark this answers is not in the document]
Comment (pahaz) [reply apps/condo/domains/acquiring/components/Alerts/NoConnectedBillings.tsx:12]: May be you need to move it inside the miniapp domain?

File: apps/condo/domains/miniapp/components/AppDescription/AboutB2BAppPage.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    })
+    const appId = get(app, 'id', null)
+    const { obj: context, error: contextError, loading: contextLoading } = B2BAppContext.useObject({
+        where: {
+            app: { id: appId },
+            organization: { id: organizationId },
+        },
+    })
+
+    const redirectUrl = `/miniapps/${id}?type=${B2B_APP_TYPE}`
```
Comment (pahaz) [inline apps/condo/domains/miniapp/components/AppDescription/AboutB2BAppPage.tsx:39]: No `id` validation before creating url

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/domains/miniapp/schema/AllMiniAppsService.js [hunk 3]
```diff
@@ -76,6 +76,28 @@ const AllMiniAppsService = new GQLCustomSchema('AllMiniAppsService', {
                     })
                 }
 
+                const B2BApps = await find('B2BApp', {
```
Comment (pahaz) [inline apps/condo/domains/miniapp/schema/AllMiniAppsService.js:79]: wrong local const naming

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/miniapp/components/AppDescription/AboutAcquiringAppPage.tsx [hunk 4]
```diff
@@ -42,14 +42,27 @@ export const AboutAcquiringAppPage: React.FC<AboutAcquiringAppPageProps> = ({ id
         },
     })
 
+    const redirectUrl = `/miniapps/${id}?type=${ACQUIRING_APP_TYPE}`
```
[the remark this answers is not in the document]
Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/components/AppDescription/AboutAcquiringAppPage.tsx:45]: Added additional check on parent level

[the remark this answers is not in the document]
Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/components/AppDescription/AboutAcquiringAppPage.tsx:45]: Fixed in parent component
https://github.com/open-condo-software/condo/pull/1540/commits/18f0289d907d2927cc33752419123b6b9a4e0d46#diff-df367fbd634530321654d967e46e900918718f3cd3cf751c4bddcc27e681d01a

File: apps/condo/domains/miniapp/components/AppDescription/AboutB2BAppPage.tsx (hunk 2 above)
Comment (SavelevMatthew) [reply apps/condo/domains/miniapp/components/AppDescription/AboutB2BAppPage.tsx:39]: Also fixed in parent component:
https://github.com/open-condo-software/condo/pull/1540/commits/18f0289d907d2927cc33752419123b6b9a4e0d46#diff-df367fbd634530321654d967e46e900918718f3cd3cf751c4bddcc27e681d01a

**Pull request merged**